qml.v1/bridge.go

```diff
@@ -1,8 +1,8 @@
 package qml
 
-// #cgo linux,amd64 CPPFLAGS: -I./cpp -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore/5.9.5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore/5.9.5/QtCore -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore/5.9.5
-// #cgo linux,arm CPPFLAGS: -I./cpp -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore/5.9.5 -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore/5.9.5/QtCore -isystem /usr/include/arm-linux-gnueabihf/qt5/QtCore/5.9.5
-// #cgo linux,arm64 CPPFLAGS: -I./cpp -isystem /usr/include/aarch64-linux-gnu/qt5/QtCore/5.9.5 -isystem /usr/include/aarch64-linux-gnu/qt5/QtCore/5.9.5/QtCore -isystem /usr/include/aarch64-linux-gnu/qt5/QtCore/5.9.5
+// #cgo linux,amd64 CPPFLAGS: -I./cpp -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore/5.12.9 -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore/5.9.5/QtCore -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore/5.12.9
```

Comment:
I know this was like this before and not the right place for this, but this is ugly xP

Reply:
It is. I'm not sure how to do this dynamically with cgo though. Something to fix later..